server/database/queries/selectScrumTasks.js

```diff
@@ -0,0 +1,14 @@
+const connect = require('./../config/connection');
+
+const selectScrumTasks = scrumId => connect.query(`SELECT
+                                                    tasks.*, users.user_name, users.role
+                                                  FROM 
+                                                    tasks
+                                                  INNER JOIN
+                                                    users
+                                                  ON
+                                                    users.id = tasks.assigned_to
+                                                  WHERE
```

Comment:
I want you to get the Scrum name as well in this query. So use another inner join with scrums